### PR TITLE
Inceptionev/issue1048 1026 v0.3 electrical documentation

### LINKED ISSUE #1048: Expose PCB assembly instructions to manufacturing/internals/display_panel/electronics/README.md
Assembly instruction content from the PCB page needs to be copied to the electronics manufacturing page for visibility.

**What has to be done**
Copy and format the instruction content from the PCB assembly page to the manufacturing page.

**How do you know it has to be done**
The way that the readmes have been restructured this content is no longer easily accessible from the general manufacturing instructions top level.

**Code pointers**
Copy content from /pcb/rev1_export to manufacturing/internals/display_panel/electronics

**Don't forget**
- [x] Put the issue in a milestone
- [x] Assign it to one of the projects
- [x] Add appropriate labels
- [x] Mention dependent/blocked issues if any

### PR DESCRIPTION
**Pull request by inceptionev**
closes #1048 
partially addresses #1026 

- Reorganizes a few READMEs to make the electronics assembly information more accessible, especially the PCB manufacturing files which will probably need to be sent to contact manufacturing vendors early in the build process.
- Adds drawings, photos, and instructions to the wiring README.
- Adds PCB stack assembly instructions and photos to the display panel assembly README.
- Adds file content descriptions to the Rev 1 PCB readme

### CODE REVIEW

File: manufacturing/internals/display_panel/electronics/README.md [hunk 1]
```diff
@@ -55,67 +57,72 @@ BEFORE purchasing any parts.**
 The ventilator assembly requires some custom wiring. Please use materials and instructions on the
 [wiring page](../../../wiring) to prepare those custom components first.
 
-### Main circuit board (PCB)
+### Mainboard (PCB)
+
+- The mainboard provides the electrtical control of the ventilator.  It is a custom-designed part that is most commonly fabricated by a PCB contract manufacturer.  It uses standard PCB manufacturing processes that can be handled by a wide range of vendors.
```
Comment (martukas) [inline manufacturing/internals/display_panel/electronics/README.md:62]: typo in "electrical"

File: manufacturing/internals/display_panel/electronics/README.md [hunk 2]
```diff
@@ -55,67 +57,72 @@ BEFORE purchasing any parts.**
...
 * real-time clock
 
-Design and manufacturing files can be found in this
-[separate repository](https://github.com/inceptionev/VentilatorRev1InterimDaughtercard).
-
-You will need to install a battery `[A3]`.
+- Design and manufacturing files can be found in this [separate repository](https://github.com/inceptionev/VentilatorRev1InterimDaughtercard).
 
-The card goes directly onto the Nucleo, **before** the stepper drivers.
+- Insert the interim controller into the sockets on the top of the Cycle Controller.  The pins will only allow one orientation of the board, so be mindful of the pin arrangement and the orientation will be correct.  Add the CR2016 battery `[A3]` for the RTC.
```
Comment (martukas) [inline manufacturing/internals/display_panel/electronics/README.md:93]: I would spell out RTC.

File: manufacturing/internals/README.md [hunk 3]
```diff
@@ -7,6 +7,11 @@
 ## Design rationale
 
 > **TODO:** Content..
+>
```
Comment (martukas) [inline manufacturing/internals/README.md:10]: I think the TODO above can go now

**Review by martukas — APPROVED** [review]
Just those small changes. Otherwise looks awesome!

File: manufacturing/internals/display_panel/electronics/README.md (hunk 1 above)
Comment (inceptionev) [reply manufacturing/internals/display_panel/electronics/README.md:62]: fixed

**Pull request merged**